pandas/tests/frame/test_indexing.py

```diff
@@ -233,44 +233,45 @@ def inc(x):
         expected = pd.DataFrame([[-1, inc], [inc, -1]])
         tm.assert_frame_equal(df, expected)
 
-    def test_getitem_boolean(self):
+    def test_getitem_boolean(self, float_string_frame, mixed_float_frame,
```

Comment:
rather than do this, either parameterize or break to smaller tests

Reply:
I get that, but again: this is only translation, reorg can happen later.

Reply:
well, I think this is worthile to do here in this pr